### DISCUSSION TITLE
Responsive utilities. Bootstrap generate only min-width.

### DISCUSSION THREAD

**Discussion by damiankomonski** [twbs/bootstrap#36957] [Q&A]
Hello!

During coding I want to use responsive utilities classes from Bootstrap. I use latest version 5.2. Unfortunately Bootstrap generate only "min-width":
https://getbootstrap.com/docs/5.2/utilities/api/#responsive

I am coding layout from desktop to mobile and use rather "max-width". Is it any option to generate classes with max-width media queries?

Here Bootstrap give media queries with max-width:
https://getbootstrap.com/docs/5.2/layout/breakpoints/#max-width

Please help. Thanks

**Comment by mdo** [comment]
You'd need to create your own mixin based on ours for that. We take a mobile first approach which assumes using `min-width` media queries.

**Reply by jenniferaslan** [reply]
If this is the case, the documentation needs to be updated to reflect this: https://getbootstrap.com/docs/5.2/layout/breakpoints/#max-width

As it is, the documentation makes it look like `@include media-breakpoint-down()` still works.

**Reply by benabraham** [reply]
Documentation is correct. OP asks about having breakpoint classes like `col-sm-3` use max-width instead of mobile-first approach.
